priv/templates/boot.eex

```diff
@@ -208,6 +208,15 @@ _replace_os_vars() {
         }1' < "$1" > "$2"
 }
 
+# Private. Run hooks from directory.
+_run_hooks_from_dir() {
+   if [ -d "$1" ]; then
+          for s in $(find "$1" -name "*.sh" -print | sort); do
```

Comment:
btw it is incorrect regexp for file path in this case, it won't match in any case, since find is returning a full path to script